### ISSUE TITLE
"Actors by Actor Role" graph in Reporting section isn't visualized

### ISSUE THREAD

**Issue by florianap** [equalitie/open-corroborator#27]
*\* tested with Iceweasel 38.7.0 on Debian 8, Firefox 45.0 on Fedora 23, Chromium 49.0.2623.87 on Debian 8.3 (64-bit) **
### Reproduction steps
1. Click on "Reporting" in the top menu of the CUI.
2. Click on "Actors".
3. Select "Actor Role" in the "Actors by" drop-down menu.
### Expected results

A graph on actor roles should appear.
### Actual results

The interface keeps showing the former graph.

![reporting_actorsbyrole](https://cloud.githubusercontent.com/assets/9517238/13848128/c1be0d4e-ec50-11e5-803e-c57c328c1ad7.png)

**Comment by graphiclunarkid** [comment]
@ggaughan, if you can reproduce this, please comment with the likely impact and the size of effort needed to fix it. We will then decide where to put it in the backlog order. If you can't reproduce it either then we should close it and wait to see if a customer finds it in the wild.

**Comment by ggaughan** [comment]
I can reproduce it. Cause is probably lack of data which fails with "Uncaught TypeError: Cannot read property 'facet_fields' of undefined". So needs some protection around that - should be a quick fix.

**Comment by florianap** [comment]
this hasn't been fixed.

**Comment by graphiclunarkid** [comment]
Yeah, we only asked @ggaughan to look into how difficult it would be to fix. We then forgot to schedule time for him to actually fix it. My bad.
